ktor-client/ktor-client-core/posix/src/io/ktor/client/engine/Loader.kt

```diff
@@ -5,9 +5,51 @@
 package io.ktor.client.engine
 
 import io.ktor.util.*
+import kotlin.native.concurrent.*
+
+private typealias T = HttpClientEngineFactory<HttpClientEngineConfig>
```

Comment:
Typealias name `T` is confusing because a usage looks like a type parameter usage